src/main/java/org/axion/axion_api_gateway/AxionApiGatewayApplication.java

```diff
@@ -30,8 +30,13 @@ public static void main(String[] args) {
 
     private static void forwardRequest(Context ctx) {
         log.info("Will forward request: {}", ctx.req().getRequestURI());
-        RequestValidator requestValidator = new RequestValidator(appConfig);
-        requestValidator.checkRequestViolations(ctx.req());
+        String requestURI = ctx.req().getRequestURI();
```

Comment:
This line can be simplified a bit.
```suggestion
        String requestURI = ctx.path();
```